src/spherely.pyi

```diff
@@ -127,12 +125,16 @@ class _VFunc_Nin1optradius_Nout1(
     @property
     def __name__(self) -> _NameType: ...
     @overload
-    def __call__(self, a: Geography, radius: float = ...) -> _ScalarReturnType: ...
+    def __call__(
```

Comment:
@benbovy One quick question about parameter name:

In the original types, the `Geography` type was called `geography` when there's only one, but `a` and `b` when there are two of them.

In the "new" `optradius` type, the `Geography` types are named with `a` also when there's only one such geography. The generator currently uses `geography` for all types when `n_in == 1`. Should I change it so that it only uses `geography` when `n_in == 1` and there are no optional args?

Reply:
I think we can leave it as-is and uniformize the argument names in a follow-up PR (Currently implemented functions with a unique geography argument use either `geography` or `a` in their docstrings).

Reply:
Yes I was a bit confused by the implementation too. Probably safer to keep raising an error in case of non-point geographies for now.

Reply:
Thanks @benbovy 

Just to make sure, should we leave this script as it is now or should I update the script to match the existing `_VFunc_Nin1optradius_Nout1`? It should be a trivial change. :)

Reply:
Using "a" even with `n_in == 1` is slightly better for now since this is the name currently used in the docstrings of the (I/O and construction) functions with one vectorized argument: https://spherely.readthedocs.io/en/latest/api.html.

Reply:
(sorry for the confusion: I had issues with posting inline comments this morning so I mixed comments from other PRs, now deleted)

Reply:
@benbovy No worries :) I've updated the script. Now the output matches the existing types.

The minor diff is because the ordering of the overloads is different for `n_in == 2`. I'm just using whatever `itertools` gives, and it's probably not worth matching the manual ordering for this.